lnd.go

```diff
@@ -430,7 +430,20 @@ func lndMain() error {
 			}
 			return delay
 		},
-		WatchNewChannel: server.chainArb.WatchNewChannel,
+		WatchNewChannel: func(channel *channeldb.OpenChannel,
+			addr *lnwire.NetAddress) error {
+
+			// First, we'll mark this new peer as a persistent
+			// re-connection purposes.
```

Comment:
(nit) comment typo "... as a persistent [peer for] re-connection purposes."

Reply:
Fixed, thanks!